src/ui/statusBarActionView.ts

```diff
@@ -0,0 +1,71 @@
+import { window, commands } from 'vscode';
+import { DafnyCommands } from '../commands';
+import * as vscode from 'vscode';
+
+export const fileIssueURL = 'https://github.com/dafny-lang/ide-vscode/issues/new?title='
++ encodeURIComponent('Verification issue')
++ '&body='
++ encodeURIComponent(`
+Failing code
+------------
+
+<!--paste relevant code below-->
+\`\`\`dafny
+
+\`\`\`
+
+Steps to reproduce the issue
+----------------------------
+
+* 
+
+Expected behavior
+-----------------
+
+
+Actual behavior
+---------------
+
+
+`.trimStart());
+
+interface StatusBarAction {
+  label: string;
+  description: string;
+  command?: string;
+  uri?: string;
+}
+
+const ReportVerificationIssue: StatusBarAction = {
+  label: 'Report verification issue',
+  description: 'File an issue on Github about a verification problem you have.',
```

Comment:
I would be careful with this; until now we have been pointing people to the "discussions" if they have trouble writing a proof

Reply:
I changed is so now the first command links to "discussions" if they need help for a proof, if verification is not successful.